package.json

```diff
@@ -55,10 +55,11 @@
     "@commitlint/config-conventional": "^8.2.0",
     "@semantic-release/changelog": "^5.0.1",
     "@semantic-release/git": "^9.0.0",
-    "axios": "^0.21.1",
+    "axios": "^0.21.4",
     "browserify": "^16.5.2",
     "codecov": "^3.8.1",
     "concat-stream": "^2.0.0",
+    "cz-conventional-changelog": "^3.3.0",
```

Comment:
Why was this dependency added?

Reply:
Also, I believe there was an ask to upgrade the ibm-cloud-sdk-core dependency to the latest version 2.14.3. While the current notation would still install the latest version we don't want users the ability to install a version of the ibm-cloud-sdk-core that still uses a vulnerable version of axios so we should be specific

Reply:
`cz-conventional-changelog` was added as a fix to the commitizen pipeline. We recommend it's use but weren't including it. It made sense to add it while we were updating the `package.json`. I got confirmation from @repjarms that it should be included. 

Ah good catch, I updated our `axios` dependency past the vulnerable version, but missed the update on the cloud core itself